### ISSUE TITLE
InvalidProxyType: 187.67.144.87:48297 is not a valid proxy type, you might be looking for [].. (RPlNWkR)

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#118]
Zeus version:
`1.0.57.977564`

Firefox version:
`(52, 4)`

Geckodriver version:
`geckodriver-v0.17.0-linux64.tar.gz`

Error info:
```Traceback (most recent call last):
  File "zeus.py", line 552, in <module>
    agent=agent_to_use, pull_all=opt.noExclude, parse_webcache=opt.parseWebcache
  File "/root/Zeus-Scanner/var/google_search/search.py", line 246, in parse_search_results
    proxy_string = proxy_string_to_dict(proxy_string)
  File "/root/Zeus-Scanner/lib/core/settings.py", line 239, in proxy_string_to_dict
    proxy_data = get_proxy_type(proxy_string)
  File "/root/Zeus-Scanner/lib/core/settings.py", line 227, in get_proxy_type
    "{}..".format(prox_list[0], difflib.get_close_matches(prox_list[0], acceptable))
InvalidProxyType: 187.67.144.87:48297 is not a valid proxy type, you might be looking for []..
````

Running details:
`Linux-4.13.0-kali1-amd64-x86_64-with-Kali-kali-rolling-kali-rolling`

Commands used:
`zeus.py --random-agent -l /root/Desktop/Queue.txt --proxy-file=/root/Desktop/url -s -B`

Log file info:
```2017-10-22 17:19:10,063;zeus-log;INFO;log file being saved to '/root/Zeus-Scanner/log/zeus-log-83.log'...
2017-10-22 17:19:10,067;zeus-log;INFO;specified to use non-default search engine...
2017-10-22 17:19:10,067;zeus-log;INFO;starting dork scan with query 'inurl:product-item.php?id='...
2017-10-22 17:19:10,067;zeus-log;WARNING;will not parse webcache URLs...
2017-10-22 17:19:10,067;zeus-log;WARNING;only pulling URLs with GET(query) parameters...
2017-10-22 17:19:10,067;zeus-log;ERROR;[7;31;31mran into exception '187.67.144.87:48297 is not a valid proxy type, you might be looking for []..'...
Traceback (most recent call last):
  File "zeus.py", line 552, in <module>
    agent=agent_to_use, pull_all=opt.noExclude, parse_webcache=opt.parseWebcache
  File "/root/Zeus-Scanner/var/google_search/search.py", line 246, in parse_search_results
    proxy_string = proxy_string_to_dict(proxy_string)
  File "/root/Zeus-Scanner/lib/core/settings.py", line 239, in proxy_string_to_dict
    proxy_data = get_proxy_type(proxy_string)
  File "/root/Zeus-Scanner/lib/core/settings.py", line 227, in get_proxy_type
    "{}..".format(prox_list[0], difflib.get_close_matches(prox_list[0], acceptable))
InvalidProxyType: 187.67.144.87:48297 is not a valid proxy type, you might be looking for []..

2017-10-22 17:19:26,946;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-22 17:19:26,947;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-22 17:19:26,984;zeus-log;INFO;[32mextracting traceback from log file...[0m
2017-10-22 17:19:26,985;zeus-log;INFO;[32mattempting to get firefox browser version...[0m
```

**Comment by Ekultek** [comment]
The proxy needs a protocol before it will work. I'll catch this error though.

**Comment by Ekultek** [comment]
So what I’m going by to do is create a way to default to `http` for the protocol, this probably will not always work, especially if you’re using something like Tor. I’ll also catch this error so that it will specify you need to pass a valid protocol.

**Comment by Ekultek** [comment]
Patched via d590ca2 it will tell you that you need to pass a protocol and will give a list of protocols
